opensquirrel/circuit_builder.py

```diff
@@ -14,7 +14,8 @@ class CircuitBuilder(GateLibrary):
     Mainly here to allow for Qiskit-style circuit construction:
 
     Example:
-        >>> CircuitBuilder(number_of_qubits=3).h(Qubit(0)).cnot(Qubit(0), Qubit(1)).cnot(Qubit(0), Qubit(2)).to_circuit()
+        >>> CircuitBuilder(number_of_qubits=3).h(Qubit(0)).cnot(Qubit(0), Qubit(1)).cnot(Qubit(0), \
```

Comment:
Have we changed anything regarding file formatting? I wouldn't use less than 120 columns for width.

Reply:
No, this line exceeded 120 columns. So that is why I shortened it.

Reply:
Cool!